CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/Conversation.java

```diff
@@ -183,17 +201,23 @@ public GraphUpdateManager getGraphUpdateManager() {
         @Override
         protected boolean update(GraphReadMethods graph) {
 
+            
             ConversationState newConversationState;
-            int conversationStateAttribute = ConversationViewConcept.MetaAttribute.CONVERSATION_VIEW_STATE.get(graph);
-            if (conversationStateAttribute == Graph.NOT_FOUND) {
-                newConversationState = new ConversationState();
-                newConversationState.setSenderAttributesToKeys(graph);
-            } else {
-                newConversationState = (ConversationState) graph.getObjectValue(conversationStateAttribute, 0);
-                if (newConversationState == null) {
+            int conversationStateAttribute;
+            if (Objects.nonNull(graph)) {
+                conversationStateAttribute = ConversationViewConcept.MetaAttribute.CONVERSATION_VIEW_STATE.get(graph);
```

Comment:
You can declare this variable here (rather than a couple lines above) since it doesn't seem to be used outside this code block. Also can make it final

Reply:
done